nodes/viz/texture_viewer.py

```diff
@@ -165,6 +161,10 @@ def __init__(self):
         items=theme_mode_options, default="default", update=updateNode
     )
 
+    def get_buffer(self,texture):
+        self.texture = texture
+        return self.texture
+
```

Comment:
@ly29 Does it correct?  or maybe you mean another thing? but i suppose this is not right...